cypress.config.js

```diff
@@ -8,12 +8,12 @@ module.exports = defineConfig({
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
```

Comment:
The `Math.random()` function should not have an argument. The correct usage is `Math.random()`, which returns a floating-point number between 0 and 1. To get a number between 1 and 1000, you should multiply by 1000 and then apply `Math.ceil()`.